assets/javascripts/discourse/templates/admin-wizards.hbs

```diff
@@ -7,8 +7,12 @@
   {{/if}}
   {{nav-item route="adminWizardsLogs" label="admin.wizard.log.nav_label"}}
   {{nav-item route="adminWizardsManager" label="admin.wizard.manager.nav_label"}}
+
+  <div class="admin-actions">
+    <a target="_blank" class="btn btn-pavilion-pro" rel="noreferrer noopener" href="https://thepavilion.io/w/support" title={{ i18n "admin.wizard.pro_support_button.title" name=wizard.pro_support_button.title}}>{{d-icon "far-life-ring"}}{{i18n "admin.wizard.pro_support_button.label" name=wizard.pro_support_button.label}}</a>
```

Comment:
@keegangeorge: style point, inconsistent space before i18n, perhaps no space in all cases?